packages/e2e-tests/helpers/testsuites.ts

```diff
@@ -1,25 +1,166 @@
-import { Locator, Page } from "@playwright/test";
+import { Locator, Page, expect } from "@playwright/test";
 
-export async function getTestRows(page: Page) {
-  return page.locator('[data-test-name="TestItemTreeRow"]');
+import {
+  clearTextArea,
+  debugPrint,
+  delay,
+  getByTestName,
+  getCommandKey,
+  mapLocators,
+  waitFor,
+} from "./utils";
+
+export function getTestSuitePanel(page: Page) {
+  return getByTestName(page, "TestSuitePanel");
+}
+
+export async function openCypressTestPanel(page: Page): Promise<void> {
+  // Only click if it's not already open; clicking again will collapse the side bar.
+  const pane = getTestSuitePanel(page);
+
+  let isVisible = await pane.isVisible();
+
+  if (!isVisible) {
+    await getByTestName(page, "ToolbarButton-CypressPanel").click();
+  }
+}
+
+export function getTestRows(page: Page) {
+  return getByTestName(page, "TestRecordingTreeRow");
 }
 
-export async function getCypressLogo(page: Page) {
-  return page.locator('[data-test-name="ToolbarButton-CypressPanel"]');
+export function getCypressLogo(page: Page) {
+  return getByTestName(page, "ToolbarButton-CypressPanel");
 }
 
-export async function getTestRowChevron(row: Locator) {
+export function getTestRowChevron(row: Locator) {
   return row.locator(":scope", { hasText: "chevron_right" });
 }
 
-export async function getTestSections(row: Locator) {
-  return row.locator('[data-test-name="TestSection"]');
+export function getTestRecordingTrees(page: Page) {
+  return getByTestName(page, "TestRecordingTree");
+}
+
+export function getTestSections(row: Locator) {
+  return getByTestName(row, "TestSection");
+}
+
+export function getTestCaseSteps(row: Page | Locator) {
+  return getByTestName(row, "TestSectionRow");
+}
+
+export function getSelectedTestCase(row: Page | Locator) {
+  return row.locator('[data-test-id="TestRecordingPanelBody"]');
+}
+
+export function getTestSuiteResult(page: Page) {
+  return getByTestName(page, "TestSuiteResult");
+}
+
+export function getTestSuiteResultsPassedCount(page: Page) {
+  return getByTestName(page, "TestSuiteResultsPassedCount");
+}
+
+export function getTestSuiteResultsFailedCount(page: Page) {
+  return getByTestName(page, "TestSuiteResultsFailedCount");
+}
+
+export function getTestSuiteResultsSkippedCount(page: Page) {
+  return getByTestName(page, "TestSuiteResultsSkippedCount");
+}
```

Comment:
IMO these methods should trim `textContent` and return numbers (or `null` if the label isn't visible)